docs/installing/ace.md

```diff
@@ -31,7 +31,7 @@ You can also set `M68K_CPU` and `M68K_FPU` variables to your liking.
 
 ``` sh
 mkdir build && cd build
-cmake .. -DCMAKE_TOOLCHAIN_FILE=/path/to/AmigaCMakeCrossToolchains/m68k.cmake -DM68K_TOOLCHAIN_PATH=/path/to/toolchain -DM68K_CPU=68000 -DM68K_FPU=soft
+cmake .. -DCMAKE_TOOLCHAIN_FILE=/path/to/AmigaCMakeCrossToolchains/m68k.cmake -DM68K_TOOLCHAIN_PATH=/path/to/toolchain -DTOOLCHAIN_PREFIX=m68k-amigaos  -DTOOLCHAIN_PREFIX_DASHED=m68k-amigaos- -DM68K_CPU=68000 -DM68K_FPU=soft
```

Comment:
You can now remove `-DTOOLCHAIN_PREFIX_DASHED=m68k-amigaos-`, also it is recommended to set `-DMCRT=nix13`